x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/oauth2/RestInvalidateTokenAction.java

```diff
@@ -54,36 +74,16 @@ public String getName() {
     @Override
     protected RestChannelConsumer innerPrepareRequest(RestRequest request, NodeClient client) throws IOException {
         try (XContentParser parser = request.contentParser()) {
-            final Tuple<String, String> tuple = PARSER.parse(parser, null);
-            final String token = tuple.v1();
-            final String refreshToken = tuple.v2();
-
-            final String tokenString;
-            final InvalidateTokenRequest.Type type;
-            if (Strings.hasLength(token) && Strings.hasLength(refreshToken)) {
-                throw new IllegalArgumentException("only one of [token, refresh_token] may be sent per request");
-            } else if (Strings.hasLength(token)) {
-                tokenString = token;
-                type = InvalidateTokenRequest.Type.ACCESS_TOKEN;
-            } else if (Strings.hasLength(refreshToken)) {
-                tokenString = refreshToken;
-                type = InvalidateTokenRequest.Type.REFRESH_TOKEN;
-            } else {
-                tokenString = null;
-                type = null;
-            }
-
-            final InvalidateTokenRequest tokenRequest = new InvalidateTokenRequest(tokenString, type);
-            return channel -> client.execute(InvalidateTokenAction.INSTANCE, tokenRequest,
-                    new RestBuilderListener<InvalidateTokenResponse>(channel) {
-                        @Override
-                        public RestResponse buildResponse(InvalidateTokenResponse invalidateResp,
-                                                          XContentBuilder builder) throws Exception {
-                            return new BytesRestResponse(RestStatus.OK, builder.startObject()
-                                    .field("created", invalidateResp.isCreated())
-                                    .endObject());
-                        }
-                    });
+            final InvalidateTokenRequest invalidateTokenRequest = PARSER.parse(parser, null);
+            return channel -> client.execute(InvalidateTokenAction.INSTANCE, invalidateTokenRequest,
+                new RestBuilderListener<InvalidateTokenResponse>(channel) {
+                    @Override
+                    public RestResponse buildResponse(InvalidateTokenResponse invalidateResp,
+                                                      XContentBuilder builder) throws Exception {
+                        invalidateResp.toXContent(builder, channel.request());
```

Comment:
given that the response is changing, this is a breaking change. I think we can have a bwc way for 6.x by keeping the "created" field as a success type thing?